addons/account/models/account_move.py

```diff
@@ -5098,7 +5097,10 @@ def _post(self, soft=True):
 
             if not invoice.partner_id:
                 if invoice.is_sale_document():
-                    validation_msgs.add(_("The field 'Customer' is required, please complete it to validate the Customer Invoice."))
+                    raise UserError(_(
+                        "The 'Customer' field is required to validate the invoice.\n"
+                        "You probably don't want to explain to your auditor that you invoiced the invisible man :)"
```

Comment:
The "the invisible man" reference will probably be lost to people who don't have it in their culture (seems to be an English/French reference, not sure if it exists in other cultures). Also it sounds awkward used this way, maybe

"that you invoiced **an** invisible man"

would work better?